code/datums/gamemodes/gm_parent.dm

```diff
@@ -194,6 +194,15 @@
 									//the first character in this string is an invisible brail character, because otherwise DM eats my indentation
 									stuff_to_output += "<b>⠀   [trace_name] (played by [trace_mind.displayed_key])<b>"
 
+							if (flock.relay_finished)
+								flock.flockmind_mind.current.unlock_medal("To the stars", TRUE)
+								boutput(flock.flockmind_mind.current, "success")
+								var/time = TIME
+								for (var/mob/living/intangible/flock/trace/flocktrace as anything in flock.traces)
+									if (time - flocktrace.creation_time >= 5 MINUTES)
```

Comment:
Not sure how I feel about the check, it might just make it more confusing why people didn't earn the medal.

Reply:
It was more to prevent cases where you make a Flocktrace last second or something when they haven't done anything. The Relay takes 6 minutes to transmit right now so should only happen if you make a Flocktrace sometime after the relay's been built.